src/tasks-check.js

```diff
@@ -0,0 +1,25 @@
+export function checking(array) {
+  array.forEach((tsk) => {
+    const checkbox = document.getElementById(`${tsk.index}-checkbox`);
+    checkbox.addEventListener('change', () => {
+      if (checkbox.checked) {
+        tsk.completed = true;
+        window.localStorage.setItem('to-do-list', JSON.stringify(array));
+      } else {
+        tsk.completed = false;
+        window.localStorage.setItem('to-do-list', JSON.stringify(array));
+      }
+    });
+  });
+}
+
+export function listchekedstat(array) {
+  array.forEach((tsk) => {
+    const checkbox = document.getElementById(`${tsk.index}-checkbox`);
+    if (tsk.completed === true) {
+      checkbox.checked = true;
+    } else {
+      checkbox.checked = false;
+    }
+  });
+}
```

Comment:
The following is just a recommendation on improvement:
- [ OPTIONAL ] You can simplify the 'listchekedstat' function by using the 'checked' attribute of the checkbox element instead of setting it to 'true' or 'false' based on the completed state.

Reply:
Ok great suggestion I'll definetly consider that. 🙏🙏🙏... Thanks!